client/ipa-client-install

```diff
@@ -1569,8 +1569,9 @@ def get_local_ipaddresses(iface=None):
             for ip in if_addrs.get(family, []):
                 try:
                     ips.append(ipautil.CheckedIPAddress(ip['addr']))
-                except ValueError:
-                    continue
+                    root_logger.debug('IP check sucessfull: %s' % ip['addr'])
+                except ValueError as e:
+                    root_logger.debug('IP check failed: %s' % str(e))
```

Comment:
I do not think we need to log success here. Btw: successful spells with two 'c' and one 'l' :)

Also, you do not need to do str(e) here as this conversion is done anyway by Python.